packages/flutter/test_fixes/cupertino.dart

```diff
@@ -115,4 +115,12 @@ void main() {
   NoDefaultCupertinoThemeData.resolveFrom(context, nullOk: false);
   CupertinoTextThemeData.resolveFrom(context, nullOk: true);
   CupertinoTextThemeData.resolveFrom(context, nullOk: false);
+
+  // Changes made in https://github.com/flutter/flutter/pull/72043
+  CupertinoTextField(maxLengthEnforced: true);
+  CupertinoTextField(maxLengthEnforced: false);
+  CupertinoTextField.borderless(maxLengthEnforced: true);
+  CupertinoTextField.borderless(maxLengthEnforced: false);
+  final CupertinoTextField textField;
+  textField.maxLengthEnforced;
```

Comment:
Isn't this going from a bool to being an enum? That'll break peoples code if we simply replace "maxLengthEnforced" with "maxLengthEnforcement", no?

Reply:
I think so, but isn't this the same as `autovalidate` -> `autovalidateMode`? bool to enum? https://github.com/flutter/flutter/pull/73996

Reply:
Oh, I didn't notice that there. It does seem a little odd that we leave your code broken after the migration.

Reply:
Discussed this offline and decided that including these partial fixes is a good way to tell people that their code needs updating for now. Will revisit if user feedback is different.